docs/ref/settings.txt

```diff
@@ -1838,21 +1838,22 @@ A list of IP addresses, as strings, that:
 
 Default: ``'en-us'``
 
-A string representing the language code for this installation. This should be in
-standard :term:`language ID format <language code>`. For example, U.S. English
-is ``"en-us"``. See also the `list of language identifiers`_ and
-:doc:`/topics/i18n/index`.
-
-:setting:`USE_I18N` must be active for this setting to have any effect.
+A string representing the fallback language code for this installation.
+This should be in standard :term:`language ID format <language code>`.
+For example, U.S. English is ``"en-us"``. See also the
+`list of language identifiers`_ and :doc:`/topics/i18n/index`.
 
-It serves two purposes:
+It serves three purposes:
 
 * If the locale middleware isn't in use, it decides which translation is served
   to all users.
 * If the locale middleware is active, it provides a fallback language in case the
   user's preferred language can't be determined or is not supported by the
   website. It also provides the fallback translation when a translation for a
   given literal doesn't exist for the user's preferred language.
+* If localization is explicitly disabled via the :tfilter:`localize` and
+  :tfilter:`unlocalize` template filters or tags. see
+  :ref:`controlling localization in templates <topic-l10n-templates>`.
 
```

Comment:
Yes, "see" needs to be capitalized and it would be good to end with "for [insert something]".  
You can just say "for more details" but you might want to be more specific like "for further explanation on [insert something]."
Rough example:
```
See :ref:`controlling localization in templates <topic-l10n-templates>` for more details.
```